EPAM_iOS_School_Practice_2_Star_Wars/EPAM_iOS_School_Practice_2_Star_Wars/Model/Person.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Person.swift
+//  EPAM_iOS_School_Practice_2_Star_Wars
+//
+//  Created by Maxim on 17.02.2020.
+//  Copyright © 2020 Maksim Svyatov. All rights reserved.
+//
+
+import UIKit
+import RealmSwift
+
+class Person: Object, Decodable {
+    @objc dynamic var name: String
+    @objc dynamic var height: String
+    @objc dynamic var mass: String
+    @objc dynamic var birthYear: String
+    @objc dynamic var gender: String
+      
+    enum CodingKeys: String, CodingKey {
+        case name
+        case height
+        case mass
+        case birthYear = "birth_year"
+        case gender
+    }
+}
+
+extension Person: Visualized {
+    var titleLabelText: String {
```

Comment:
Very very bad approach, strong connection between data.
If there are 4 parameters, what will happen?
Try to use enum, or something of this

```
enum PersonData {
   case height(label: String, value: String)
   case ...
}
```

Reply:
Fixed, but without enum